Framework/Core/CMakeLists.txt

```diff
@@ -0,0 +1,78 @@
+# Copyright CERN and copyright holders of ALICE O2. This software is
+# distributed under the terms of the GNU General Public License v3 (GPL
+# Version 3), copied verbatim in the file "COPYING".
+# 
+# See https://alice-o2.web.cern.ch/ for full licensing information.
+# 
+# In applying this license CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+set(MODULE_NAME "Framework")
+
+set(MODULE_BUCKET_NAME O2DeviceApplication_bucket)
```

Comment:
Probably that should be a separate `FrameworkCore_bucket`. The linker error comes from the missing `libNet` from ROOT where the TMessage is implemented.

Reply:
Yes, agreed. I will fix it.